### DISCUSSION TITLE
How to switch catagory to `R1` and `R2` in Vem mode?

### DISCUSSION THREAD

**Discussion by flyingicedragon** [garywill/BigSearch#18] [Q&A] [answered]
I saw a `R1` and a `R2` remind for catagory when I enter Vem mode. However, which key should I use to switch?

**Comment by garywill** [comment, marked as answer]
Press R then press 1. Simple

**Reply by flyingicedragon** [reply]
Thanks for your help! I tried `R` with `Shift` but I didn't think to try this😓
